package.json

```diff
@@ -143,6 +143,10 @@
   "lint-staged": {
     "*.js|{lib,setup,bin,hot,buildin,tooling,schemas}/**/*.js|test/*.js|{test,examples}/**/webpack.config.js}": [
       "eslint --cache"
+    ],
+    "*.md": [
+      "prettier --write",
+      "git add"
```

Comment:
I don't like precommit checks to magically modify the files. Could you change this to only `prettier` without `git add` or `--write`.

Reply:
I try without `git add` or `--write`, fell weird, format change file not staged, this will cause the developer to don't know what happened, not good